pkg/generate/app/templatelookup.go

```diff
@@ -92,7 +100,8 @@ func (r *TemplateFileSearcher) Search(terms ...string) (ComponentMatches, error)
 			Object()
 
 		if err != nil {
-			return nil, err
+			glog.V(5).Infof("Error from template search: %v", err)
+			continue
```

Comment:
Hrm, maybe.  I'm not happy with the error handling here.

On Thu, Jan 21, 2016 at 12:02 PM, Ben Parees <notifications@github.com>
wrote:

> In pkg/generate/app/templatelookup.go
> <https://github.com/openshift/origin/pull/6670#discussion_r50429491>:
>
> > @@ -92,7 +100,8 @@ func (r *TemplateFileSearcher) Search(terms ...string) (ComponentMatches, error)
> >  			Object()
> >
> >  		if err != nil {
> > -			return nil, err
> > +			glog.V(5).Infof("Error from template search: %v", err)
> > +			continue
>
> not aborting on an error seems like a mistake. or at least printing it as
> a warning. Doesn't this mean if the template file the user intended to use
> has a typo, you're going to silently skip over that file?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/6670/files#r50429491>.
>